Essentials/src/main/resources/messages.properties

```
@@ -1333,6 +1357,10 @@ unlimitedCommandUsage3=/<command> clear [player]
 unlimitedCommandUsage3Description=Clears all unlimited items for yourself or another player if specified
 unlimitedItemPermission=\u00a74No permission for unlimited item \u00a7c{0}\u00a74.
 unlimitedItems=\u00a76Unlimited items\:\u00a7r
+unlinkCommandDescription=Unlinks your minecraft account from any associated discord account.
+unlinkCommandUsage=/<command>
+unlinkCommandUsage1=/<command>
+unlinkCommandUsage1Description=Unlinks your minecraft account from any associated discord accounts.
```

```suggestion
unlinkCommandDescription=Unlinks your Minecraft account from the currently-linked Discord account.
unlinkCommandUsage=/<command>
unlinkCommandUsage1=/<command>
unlinkCommandUsage1Description=Unlinks your Minecraft account from the currently-linked Discord account.
```

TODO one-to-one one-to-many-to-many-to-one-to-wat